tenant/service_bucket.go

```diff
@@ -108,35 +108,38 @@ func (s *Service) FindBuckets(ctx context.Context, filter influxdb.BucketFilter,
 		return nil, 0, err
 	}
 
-	needsSystemBuckets := true
-	for _, b := range buckets {
-		if b.Type == influxdb.BucketTypeSystem {
-			needsSystemBuckets = false
-			break
+	// NOTE(affo): why should we artificially append system buckets if they are not there?
```

Comment:
yeah, why?

Reply:
this is a remnant of the old system. there are old org's that dont have system buckets stored but still need to be displayed.

Reply:
mmmm ok, I had a failing test, because that way asking for buckets always return them.
Maybe I can fix the test if this has to stay here 👍